src/components/pages/CaseOverview/CaseDetails.js

```diff
@@ -49,18 +48,53 @@ const CaseDetails = props => {
         <Icon component={() => <img src={OrangeLine} alt="divider icon" />} />
       </p>
       <div>
-        <p>{`Case Date: ${converted}`}</p>
-        <p>{`Judge Name: ${caseData.first_name} ${caseData.middle_initial}. ${caseData.last_name}`}</p>
-        <p>{`Origin City: ${caseData.case_origin_city}`}</p>
-        <p>{`Origin State: ${caseData.case_origin_state}`}</p>
-        <p>{`Gender: ${caseData.gender}`}</p>
-        <p>{`Type of Persecution: ${caseData.type_of_persecution}`}</p>
-        <p>{`Indigenous Group: ${caseData.indigenous_group}`}</p>
-        <p>{`Protected Grounds: ${caseData.protected_grounds}`}</p>
-        <p>{`Outcome: ${caseData.outcome}`}</p>
-        <p>{`Country of Origin: ${caseData.country_of_origin}`}</p>
-        <p>{`Filed Within One Year: ${caseData.check_for_one_year}`}</p>
-        <p>{`Applicant Language: ${caseData.applicant_language}`}</p>
+        <p id="top_item_case_details_item" class="case_details_item">
+          <span class="styled_case_details">Case Date:</span> {`${converted}`}
+        </p>
+        <p class="case_details_item">
+          <span class="styled_case_details">Judge Name:</span>
+          {` ${caseData.first_name} ${caseData.middle_initial}. ${caseData.last_name}`}
```

Comment:
Just curious what does the . mean after ${caseData.middle_initial}. Does it have a purpose or was it a typo? If it has a purpose could you explain what it does?

Reply:
I believe this is so there is a . after the middle initial (i.e. Jenna D. Anderson) since it's an abbreviation

Reply:
Thank you for pointing this out, attention to detail is really important here. In this case, this period exists to denote a middle initial.